metrics/codahale.go

```diff
@@ -225,6 +225,8 @@ func (c *CodaHale) IncErrorsStreaming(routeId string) {
 	}
 }
 
+func (c *CodaHale) ObserveRouteKubeResources(ingress string, routeId string, service string) {}
```

Comment:
this could be shortened into:
```suggestion
func (c *CodaHale) ObserveRouteKubeResources(ingress, routeId, service string) {}
```